src/actions/clusterActions.js

```diff
@@ -452,15 +452,17 @@ export function clusterLoadDetails(
     } catch (error) {
       if (error.status === StatusCodes.NotFound) {
         new FlashMessage(
-          'This cluster no longer exists.',
+          `Cluster <code>${clusterId}</code> no longer exists.`,
           messageType.INFO,
-          messageTTL.MEDIUM,
-          'Redirecting you to your organization clusters list'
+          messageTTL.MEDIUM
         );
```

Comment:
Removed this because this message can appear in cluster list, which doesn't make sense at all.